Plotted the rmse and nse

RAtshan opened the pull request:
Plotting the rmse and nse to compare the errors for the hybrid and process models. We distinguished based on the reach type. Within the stream, we distinguished based on the number of observations for each reach.

Commit:
Plotted the rmse and nse to compare the errors for the hybrid and process models. We distinguished based on the reach type. Within the stream, we distinguished based on the number of observations for each reach.

limnoliver commented on code/Metric_visualization.R:
```diff
@@ -0,0 +1,66 @@
+  geom_abline(intercept = 0, slope = 1, linetype = "dashed") +
+  #geom_smooth(method = "lm") +
+  scale_x_continuous(limits = c(0,9), breaks = 0:9) +
+  scale_y_continuous(limits = c (0, 7), breaks = 0:7)+
```
With this kind of comparison, it's often nice to have the same limits on the x and y axes so you can see the true shift from the 1:1 line.

limnoliver commented on code/Metric_visualization.R:
```diff
@@ -0,0 +1,66 @@
+stream_type <- readRDS('data/stream_type .RDS')
+stream_type$seg_id_nat <- as.character(stream_type$seg_id_nat)
+
+dot_size <- rgnc_dat_filter %>%
```
This is okay for now because we're not doing anything formal (so no need to make changes), but usually we want to make each script reproducible. Currently, `rgcn_dat_filter` and `rmse` and `nse_metric` can't be reproduced from this script. If I wanted to run these locally, I'd have to find the script that produces thoses dataframes.

limnoliver commented on code/Metric_visualization.R:
```diff
@@ -0,0 +1,66 @@
+    scale_color_manual(values = c("grey27", "red")) + 
+    theme(plot.title = element_text(hjust =  0.5, size = 20, face = "bold"),
+          axis.title = element_text(size = 14, face = "bold"))
+nse_plot
```
can you provide a function at the end that writes the plots? I like to use `ggsave`

limnoliver commented:
This is looking really good, Rasha!

Commit:
Line 39: used the same axis limit to compare the process model vs hybrid model.

Produced and pushed rmse and nse plots to compare the process model vs hybrid model.

Commit:
Adjusting the rmse & nse plot size.

Commit:
Changes
